cpp/src/join/joining.cu

```diff
@@ -604,91 +632,55 @@ gdf_error join_call_compute_df(
     }
 
     CHECK_STREAM(0);
-
-
-    return df_err;
+    
```

Comment:
Can't comment on them since they are collapsed, but can you eliminate some of the unnecessary repeated blank lines above?